appIcons.js

```diff
@@ -443,6 +445,61 @@ const MyAppIcon = new Lang.Class({
         Clutter.cairo_set_source_color(cr, bodyColor);
         cr.fill();
         cr.$dispose();
+    },
+
+    _numberOverlay: function() {
+        // Add label for a Hot-Key visual aid
+        this._numberOverlayLabel = new St.Label();
+        this._numberOverlayBin = new St.Bin({child: this._numberOverlayLabel });
```

Comment:
It would be nice if we could add a style here, so that it is easy to edit from stylesheet.css.
I'm not sure how to do it though.
This is not strictly necessary of course, just an extra feature.